adoc/deployment-terraform-example.adoc

```diff
@@ -283,3 +283,127 @@ plane instance. Leave empty to have {tf} create it.
 instances. Leave empty to have {tf} create it.
 
 # end::tf_aws[]
+
+[[tf.azure]]
+=== Terraform Azure Configuration
```

Comment:
Please help to sync the configuration example with [Fix typo and rewording for Azure terraform.tfvars.example](https://github.com/SUSE/skuba/pull/1203#pullrequestreview-438859058)

Reply:
changed